elastic/kibana
# [Synthetics] adjust overview status to account for various filters

## Summary

Resolves https://github.com/elastic/kibana/issues/152528
Resolves https://github.com/elastic/kibana/issues/152525
Resolves https://github.com/elastic/kibana/issues/152290

Summarize your PR. If it involves visual changes include a screenshot or gif.

- Adjusts overview status query to take into account location filter for pending and disabled count
- Adjusts overview status front end logic to take into account status filter for pending count 

[Synthetics-Overview-Synthetics---Kibana (6).webm](https://user-images.githubusercontent.com/11356435/222312223-a529e709-8171-4721-aeba-5dcbf23949a2.webm)

### Testing

1. Create two monitors, each with locations set to `North America - Central`, `US Central Staging` and and `US Central QA`.
2. Disable one of the monitors
3. Utilize the quick `up` `down` and `disabled` filters. Ensure the counts are correct.
4. Filter by locations, ensure the pending and disabled counts are correct
